tests/src/a1TestFiles.cpp

```diff
@@ -0,0 +1,348 @@
+#include "a1TestFiles.h"
+
```

Comment:
This file should be part of gitignore since it keeps getting regen'd. So keep this out of the repo